index.html

```diff
@@ -0,0 +1,266 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>ae-HTML-Quiz</title>
+  <link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+<!-- Div wrapping the elements contained in the body -->
+  <div class="wrapper">
+    <article class="card">
+
+      <!-- card header -->
+      <header class="header">
+        <h1>Playing Cards</h1>
```

Comment:
You will want to change where the border is. Instead of putting the border around the `<h1>` where could you put it to make it reflect the example? Also you want the border to be dotted.